velox/common/memory/tests/MemoryArbitratorTest.cpp

```diff
@@ -591,15 +595,16 @@ TEST_F(MemoryReclaimerTest, orderedReclaim) {
   // child and 4 from 4th child and 1 from 2nd.
   // So expected reclaimable allocation units are {0, 0, 2, *0*, *0*}
   ASSERT_EQ(
-      root->reclaimer()->reclaim(root.get(), 10 * allocUnitBytes),
+      root->reclaimer()->reclaim(root.get(), 10 * allocUnitBytes, stats_),
```

Comment:
Can we have a fake test here to verify the stats has been bumped and aggregated? Thanks!